tests/Console/ImportGroupsCommandTest.php

```diff
@@ -5,18 +5,36 @@
 use Tests\TestCase;
 use Rogue\Models\GroupType;
 
-class ImportRhodeIslandGroupsCommandTest extends TestCase
+class ImportGroupsCommandTest extends TestCase
 {
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        config(['import.group_types.test' => [
+            'filter_by_location' => true,
+            'name' => 'Automated Test Group Type',
+            'path' => 'tests/Console/example-groups.csv',
+        ]]);
+    }
+
+    public function tearDown(): void
+    {
+        parent::tearDown();
+
+        config(['import.group_types.test' => null]);
```

Comment:
What's bizarre is that I do get a `tearDown` working with `Config::set` over in https://github.com/DoSomething/chompy/blob/b6f9003f5e4f04f39a9740a30ab6235c069bd959/tests/Unit/Models/RockTheVoteReportTest.php#L29-L36

But when I edit this line to
```
 \Config::set('import.group_types.test', false);
``` 

It results in this error:
```

1) Tests\Console\ImportGroupsCommandTest::testImportingGroups
Error: Call to undefined method Illuminate\Support\Facades\Config::set()
```

Reply:
I think the problem you're running into here is that the application has already been "torn down" and so all the services that the application relies on (like `Config`) have been destroyed. You could sidestep this error by moving this line up above the `parent::tearDown()` call (before that method destroys the config service), and it'd no longer error out but we can actually do one better...

Because the config service (and the whole rest of the application's state) is destroyed and re-built in between every single test, you actually don't even need to reset this at all! Each test method will, by default, get a brand new version of the app (based on the normal config, test environment variables, and anything you do in `setUp`).

You should be safe to remove `tearDown` here entirely! 🔥